src/compositor/mod.rs

```diff
@@ -57,7 +57,7 @@ impl Compositor {
     /// Poses are relative to the origin set by `set_tracking_space`.
     pub fn wait_get_poses(&self) -> Result<WaitPoses, CompositorError> {
         unsafe {
-            let mut result: WaitPoses = mem::uninitialized();
+            let mut result: WaitPoses = mem::MaybeUninit::uninit().assume_init();
```

Comment:
If we're going to adopt `MaybeUninit`, we should so so correctly, i.e. don't call `assume_init` until *after* the memory has been initialized.